### PR TITLE
Ansiha ckpt perf cpu

### PR DESCRIPTION
**Pull request by A9isha**

### CODE REVIEW

File: MaxText/llama_or_mistral_ckpt.py [hunk 1]
```diff
@@ -41,6 +41,7 @@
 from train import save_checkpoint
 import torch
 import sys
+import os
```
Comment (rwitten) [inline MaxText/llama_or_mistral_ckpt.py:44]: remove

File: MaxText/llama_or_mistral_ckpt.py [hunk 2]
```diff
@@ -394,4 +417,8 @@ def convert(base_model_path, maxtext_model_path, model_size):
 
   if args.model_size not in MODEL_PARAMS_DICT:
     raise NotImplementedError
+
+  if args.model_size == "llama2-70b":
```
Comment (rwitten) [inline MaxText/llama_or_mistral_ckpt.py:421]: probably sharding is always welcome?

**Review by rwitten — APPROVED** [review]
One little nit.

But more substantively I think the two paths (one sharded, one unsharded) is bad because it increases maintenance burden. I think one path is right?

File: MaxText/llama_or_mistral_ckpt.py (hunk 1 above)
Comment (A9isha) [reply MaxText/llama_or_mistral_ckpt.py:44]: we need this for enabling the device simulation flag
`os.environ['XLA_FLAGS'] = f'--xla_force_host_platform_device_count={SIMULATED_CPU_DEVICES_COUNT}'`

File: MaxText/llama_or_mistral_ckpt.py (hunk 2 above)
Comment (A9isha) [reply MaxText/llama_or_mistral_ckpt.py]: Done - thanks!

**Pull request merged**